packages/main/src/themes/base/Popover.less

```diff
@@ -92,4 +92,8 @@
 
 .sapMPopoverArr.sapMPopoverArrHidden {
 	display: none;
+}
+
+.sap-ios .sapMPopover {
```

Comment:
Please check if needed on Android as well.

Reply:
> Please check if needed on Android as well.
Cheked, the issue seems to on iOS